server/app/lib/actions/fusor/deployment/rhev/ose_launch.rb

```diff
@@ -28,9 +28,43 @@ def plan(deployment)
           def run
             ::Fusor.log.debug '====== OSE Launch run method ======'
             deployment = ::Fusor::Deployment.find(input[:deployment_id])
-            compute_attrs = create_compute_profile(deployment).vm_attrs
-            host = create_host(deployment, compute_attrs)
-            #TODO save host.ip and host.name in deployment
+
+            # launch master nodes
+            for i in 1..deployment.openshift_number_master_nodes do
+              vmlauncher = Utils::Fusor::VMLauncher.new(deployment, 
+                                                        'ose', 
+                                                        deployment.openshift_install_loc,
+                                                        'RedHat 7.1',
```

Comment:
Instead of hardcoding the RedHat 7.1, it'd be better if we were reading this from a setting in fusor.yaml.